cfme/containers/provider/__init__.py

```diff
@@ -351,7 +352,8 @@ class Edit(CFMENavigateStep):
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        sel.check(Quadicon(self.obj.name, self.obj.quad_name).checkbox())
+        self.prerequisite_view.entities.get_entity(self.obj.name,
```

Comment:
I guess it would be better to call Details and click appropriate configuration item in step
`prerequisite = NavigateToSibling('Details')`
and
`view.configuration.item_select('blabla')`

Reply:
about `view.configuration.item_select('blabla')`  -  I would like this change in different PR.